fhir-bulkdata-webapp/src/main/java/com/ibm/fhir/bulkdata/jbatch/context/BatchContextAdapter.java

```diff
@@ -133,6 +133,7 @@ public BulkDataContext getStepContextForImportChunkReader() {
         context(ctx);
```

Comment:
Update copyright date.

Reply:
I have it in my other open PR. I'm going to defer this, as it'll fix when that is merged.

Reply:
PR https://github.com/IBM/FHIR/pull/3393 is the one that has it